tests/cases/rest_decorator_test.py

```diff
@@ -47,18 +54,31 @@ def pointless_endpoint_unicode(self, path, params):
     def pointless_endpoint_bytes(self, path, params):
         raise Exception('\x80\x80 cannot be converted to unicode or ascii.')
 
-    def test_endpoint_exception_ascii(self):
+    def tes_endpoint_exception_ascii(self):
```

Comment:
I turned those off to speed up my test development and forgot to turn them back on. Good catch :)